lib/bank_account.rb

```diff
@@ -0,0 +1,29 @@
+class BankAccount
+ 
+  attr_accessor :name, :job, :email, :address, :balance 
+
+  def initialize(name, job, email, address)
+    @name = name
+    @job = job
+    @email = email 
+    @address = address 
+    @balance = 0
+  end
+
+  def deposit_money(money)
+    @balance += money 
+    puts "Deposited $ #{money}. Your account balance is $ #{@balance}."
+  end
+
+  def withdraw_money(money)
+    if money > 5000
```

Comment:
The assignment was to check that in 1 day all withdrawals cannot pass 5000$. If I understand correctly, using your approach, I will be able to extract 4.999$ multiple times in the same day.

Reply:
I think it would be a good idea to also add a test for this use case